tests/tshirts_test.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import os
+import sys
+
+# Add the parent directory to path
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+from tshirts_alternate import size
+
+class TestTShirtSizing(unittest.TestCase):
+    def test_small_size(self):
+        self.assertEqual(size(37), 'S')
+
+    def test_medium_size(self):
```

Comment:
You're good at exploring all of the data space. One improvement - try wording the tests by their expectations as well. For example, instead of `test_boundary_case`, you could have `test_boundary_gives_lose_fit` for the 38 being 'M'